apps/rush-lib/src/api/CommandLineConfiguration.ts

```diff
@@ -56,19 +96,53 @@ export class CommandLineConfiguration {
           }
         }
       }
-
     }
   }
 
   /**
-   * Loads the configuration from the specified file.  If the file does not exist,
-   * then an empty default instance is returned.  If the file contains errors, then
-   * an exception is thrown.
+   * Loads the configuration from the specified file and applies any omitted default build
+   * settings.  If the file does not exist, then an empty default instance is returned.
+   * If the file contains errors, then an exception is thrown.
    */
   public static loadFromFileOrDefault(jsonFilename: string): CommandLineConfiguration {
     let commandLineJson: ICommandLineJson | undefined = undefined;
     if (FileSystem.exists(jsonFilename)) {
-      commandLineJson = JsonFile.loadAndValidate(jsonFilename, CommandLineConfiguration._jsonSchema);
+      commandLineJson = JsonFile.load(jsonFilename);
+
+      // merge commands specified in command-line.json and default (re)build settings
+      // Ensure both build commands are included and preserve any other commands specified
+      if (commandLineJson && commandLineJson.commands) {
+        for (let i: number = 0; i < commandLineJson.commands.length; i++) {
+          const command: CommandJson = commandLineJson.commands[i];
+
+          // Determine if we have a set of default parameters
+          let commandDefaultParameters: CommandJson | {} = {};
+          switch (command.commandKind) {
+            case RushConstants.bulkCommandKind: {
+              switch (command.name) {
+                case RushConstants.buildCommandName: {
+                  commandDefaultParameters = CommandLineConfiguration.defaultBuildCommandJson
```

Comment:
"Parameters" has a special meaning of command-line parameter definition.

Suggest to rename `commandDefaultParameters` to `commandDefaultDefinition`.